### ISSUE TITLE
Errors while reading "the answer"

### ISSUE THREAD

**Issue by Stasik0** [acplt/open62541#183]
Maybe an another regression: while reading "the answer" i get:
```
NodeClass has bad statuscode: BadNodeIdUnknown
DisplayName has bad statuscode: BadNodeIdUnknown
BrowseName has bad statuscode: BadNodeIdUnknown
Read succeeded.
Reading type info of NodeId NS0|Numeric|6
```
It seems that UaExpert wants to browse node 6 which is Int32, wontfix would be okay for me

**Comment by jpfr** [comment]
Nodeid(ns=0, i=6) is the int32 datatype.

In ua_services_attribute we return this data.
Where do you get the error messages?

**Comment by Stasik0** [comment]
it seems to be fixed with 9b03e6bd0a22236a99d23fa3c2a5981a0fa0f315, can anyone confirm?

**Comment by jpfr** [comment]
Confirmed
